include/fs_utils.h

```diff
@@ -168,12 +199,31 @@ std::deque<std_fs::path> get_xdg_data_dirs() noexcept;
 
 union FatAttributeFlags; // forward declaration
 
-FILE* local_drive_create_file(const std_fs::path& path,
-                              const FatAttributeFlags attributes);
+NativeFileHandle local_drive_open_file(const std_fs::path& path,
+                                       const bool write_access);
+NativeFileHandle local_drive_create_file(const std_fs::path& path,
+                                         const FatAttributeFlags attributes);
 uint16_t local_drive_create_dir(const std_fs::path& path);
 uint16_t local_drive_get_attributes(const std_fs::path& path,
                                     FatAttributeFlags& attributes);
 uint16_t local_drive_set_attributes(const std_fs::path& path,
                                     const FatAttributeFlags attributes);
 
+// Windows ReadFile and WriteFile take in a DWORD (32 bit unsigned) for number of bytes
+// We'll make that our limit until someone can give a good reason why they need to read/write more than 4GB (unlikely)
```

Comment:
Indeed 😎

Reply:
@johnnovak And the maximum file size for FAT32 is still 4GB.

Reply:
Removed this comment.  It can handle large files now.